### DISCUSSION TITLE
bag update - 25Q1

### DISCUSSION THREAD

**Discussion by leondz** [NVIDIA/garak#881] [General]
probes
* include latentinjection
* include suffix
* include tap cached
* include autodan cached
* include ansiescape
* remove dans except daninthewild,danablation
* include all encoding
* ensure new packagehallucination items are there
* move knownbadsignatures to av_spam_scanning
* remove donotanswer
* enable extended detectors
* consider only using tier1,2 probes?

models
* nous?
* long context
* jamba
* palmyra

**Comment by leondz** [comment]
tier 1 + 2 probes:
```
plugins:
    probe_spec: atkgen.Tox,continuation.ContinueSlursReclaimedSlurs,dan.Ablation_Dan_11_0,dan.AutoDANCached,dan.DanInTheWild,encoding,fileformats.HF_Files,goodside.Tag,goodside.ThreatenJSON,goodside.WhoIsRiley,grandma,latentinjection,leakreplay,lmrc.Bullying,lmrc.Deadnaming,lmrc.QuackMedicine,lmrc.SexualContent,lmrc.Sexualisation,lmrc.SlurUsage,malwaregen,misleading.FalseAssertion50,packagehallucination,promptinject,realtoxicityprompts.RTPBlank,replay.Repeat,snowball.GraphConnectivityMini,snowball.SenatorsMini,suffix.GCGCached,tap.TAPCached,xss.MarkdownImageExfil
    extended_detectors: true
```

* ensure mini and non-mini versions are all accounted for
* do we want to add.. BEAST? AutoDAN? TAP? GCG? yeah I guess, what's fast

**Comment by erickgalinkin** [comment]
IMO, we should run everything Tier 1 + Tier 2 that is `active`. 
I'd ideally like to have a bag of ~20 models but that seems steep.

**Comment by leondz** [comment]
AILuminate (the ML Commons safety benchmark) have a cutoff that's "the two best models under 15B", the size being there to make it about "reasonably accessible" models. This sounds like an OK thing for us to measure in the bag - any objections around adding it to to calibration as another metric? 

They use it to delineate between good and very good, ie between the top two and bottom three in their five-category benchmark. Maybe we could call it "competitive_accessible" 
![IMG_20250211_113341.jpg](https://github.com/user-attachments/assets/69688e11-6665-45d3-acd6-cc3fe4ce1c14)
